### PR TITLE
Compatibility with OpenId implementations

### PR DESCRIPTION
**Pull request by aschekatihin**
that do not follow '/.well-known/jwks.json' url schema

### CODE REVIEW

**Comment by aschekatihin** [comment]
Added test.
Also are tests working for you? For me it does nothing and quits.

image

For me script like this works: ```"test": "tape -r ts-node/register __tests__/**/*.{js,ts}"```

**Comment by Levino** [comment]
See CI: https://circleci.com/gh/Levino/mock-jwks/24?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

All tests are run.

File: package.json [hunk 1]
```diff
@@ -55,7 +55,8 @@
     "ts-jest": "^22.4.1",
     "ts-node": "^7.0.1",
     "tslint": "^5.9.1",
-    "typescript": "^2.7.2"
+    "typescript": "^2.7.2",
+    "yarn": "^1.13.0"
```
Comment (Levino) [inline package.json]: That is not necessary. You should use `yarn` instead of `npm`.

**Comment by Levino** [comment]
I will take your work and make a new PR based on it because I want to take a slightly different approach.

**Comment by Levino** [comment]
Superseded by #10

**Pull request closed without merging**